examples/for-tests/src/App.js

```diff
@@ -746,6 +746,14 @@ function getPasswordlessConfigs({ disableDefaultImplementation }) {
             disableDefaultImplementation,
             style: theme.style,
         },
+        translations: {
```

Comment:
We should consider changing the name to `translationsForLanguages`

Reply:
If this name is changing, we should also change the name of the functions in the index.ts files